.changes/unreleased/tinymce-TINY-10681-2024-02-22.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: Added new `license_key` option.
```

Comment:
I feel like we need more detail, but I'm not sure what to say 🤔

Reply:
I changed it to "Added new `license_key` option that can be set to `gpl` or a valid license key." the release notes and docs will have to be more flushed out here.